### ISSUE TITLE
Precinct.ElectoralDistrictIds, Precinct.PrecinctSplitName, and Precinct Example Documentation Update

### ISSUE THREAD

**Issue by jungshadow** [votinginfoproject/vip-specification#325]
[In the documentation](http://vip-specification.readthedocs.io/en/latest/built_rst/xml/elements/precinct.html) of `Precinct.ElectoralDistrictIds` it references `Precinct.PrecinctSplitName` and states:

> If an electoral district splits a precinct, use the PrecinctSplitName object and do not specify that particular electoral district in this object.

I thought that, from our conversations on #91, if `Precinct.PrecinctSplitName` was specified, it meant that the `Precinct` element was a precinct split, which makes this a bit confusing. After re-reading #27, it sounds like the idea is that every `Precinct` element is a precinct split unless `Precinct.PrecinctSplitName` is specified (_**NB:** how does this handle multiple splits within a precinct if `Precinct.PrecinctSplitName` can only be specified once?_) If I'm interpreting the documentation incorrectly–or my understanding of the overall concept is flawed–it would be helpful to clarify the language in those two sub-elements and add another example showing the use of `Precinct.PrecinctSplitName`.

**Comment by jdmgoogle** [comment]
This is definitely confusing, thanks for pointing it out.

The basic gist is:

* If `PrecinctSplitName` is empty, then this object represents a full precinct.
* If `PrecinctSplitName` is not empty, then this represents a portion (or split) of a precinct.
* If a precinct is split into *N* parts, then there will be *N* `Precinct` objects which have the same `PrecinctName` but different `PrecinctSplitName`s.

Does this help clarify? Once we get the wording down I'll update the documentation.

**Comment by jungshadow** [comment]
@jdmgoogle Ah...that makes a lot of sense. Thanks! So to make a quick-and-ugly example (_**NB:** idea is `ElectoralDistrict` elements ed60129 and ed60037 split `Precinct` pre90111_):

```xml
<Precinct id="pre90111">
   <ElectoralDistrictIds>ed60311 ed60054</ElectoralDistrictIds>
   <IsMailOnly>false</IsMailOnly>
   <LocalityId>loc70001</LocalityId>
   <Name>203 - GEORGETOWN</Name>
   <Number>0203</Number>
   <PollingLocationIds>pl81274</PollingLocationIds>
</Precinct>

<Precinct id="pre90112">
   <ElectoralDistrictIds>ed60129</ElectoralDistrictIds>
   <IsMailOnly>false</IsMailOnly>
   <LocalityId>loc70001</LocalityId>
   <Name>203 - GEORGETOWN</Name>
   <PrecinctSplitName>203A - Georgetown</PrecinctSplitName>
   <Number>0203A</Number>
   <PollingLocationIds>pl81274</PollingLocationIds>
</Precinct>

<Precinct id="pre90113">
   <ElectoralDistrictIds>ed60037</ElectoralDistrictIds>
   <IsMailOnly>false</IsMailOnly>
   <LocalityId>loc70001</LocalityId>
   <Name>203 - GEORGETOWN</Name>
   <PrecinctSplitName>203B - Georgetown</PrecinctSplitName>
   <Number>0203B</Number>
   <PollingLocationIds>pl81274</PollingLocationIds>
</Precinct>
```

Does this seem semi-accurate?

**Comment by jdmgoogle** [comment]
There are actually examples in the sample feed. E.g.,

https://github.com/votinginfoproject/vip-specification/blob/vip52/sample_feed.xml#L1735

I'll just update the documentation to be more clear about how it's supposed to work.

**Comment by jungshadow** [comment]
@jdmgoogle I've been gone too long... I didn't even think to check there :) I think it would be helpful to lift a simple example out of the sample feed into the docs, but I notice you now have a link to it in the PR, so that may be enough.

**Comment by jdmgoogle** [comment]
Closed by #344.
